Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery/Definitions/Result.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Frends.Oracle.ExecuteQuery.Definitions
+{
+    /// <summary>
+    /// Return object with private setters.
+    /// </summary>
+    public class Result
+    { 
+        /// <summary>
+        /// Boolean value of success of the query 
+        /// </summary>
+        /// <example>true</example>
+        public bool Success { get; set; }
```

Comment:
This should have private setter.

Reply:
Is there a way to use private setter in a class that is being inherited. When I tried that it gave errors: 
"Error CS0272 The property or indexer 'Result.Success' cannot be used in this context because the set accessor is inaccessible.". Should this be protected then?

Reply:
Internal setter seems to be working.

Reply:
I also got it working with protected setter. And with some investigating the protected is same as private but it is visible to derived classes.